development/docker-compose.yml

```diff
@@ -47,8 +53,10 @@ services:
         - "-c"
         - "nautobot-server celery inspect ping --destination celery@$$HOSTNAME"
     depends_on:
-      - nautobot
-      - redis
+      nautobot:
+        condition: service_healthy
```

Comment:
This keeps the `celery_worker` container from attempting to startup while the main `nautobot` container is still in the midst of running initial migrations.